lib/workers/bin/run_single_worker.rb

```diff
@@ -48,7 +52,7 @@
 opt_parser.abort(opt_parser.help) unless worker_class
 
 unless ::MIQ_WORKER_TYPES.keys.include?(worker_class)
-  puts "ERR:  `#{worker_class}` WORKER CLASS NOT FOUND!  Please run with `-l` to see possible worker class names."
+  STDERR.puts "ERR:  `#{worker_class}` WORKER CLASS NOT FOUND!  Please run with `-l` to see possible worker class names."
```

Comment:
Thanks for fixing this (changing to `STDERR`).  I probably should have done this in the first place when I added this, but was lazy.  👍